src/dns.mli

```diff
@@ -791,6 +791,54 @@ module Edns : sig
       buffer. *)
 end
 
+(** Loc
+
+    A locator record (LOC) is used to express location information associated with domain.
+    
+    See RFC 1876. *)
+module Loc : sig
+  type t = {
+    latitude : int32;
+    longitude : int32;
+    altitude : int32;
+    size : int;
+    horiz_pre : int;
+    vert_pre : int;
+  }
+  (** The type of a Loc record. *)
+
+  val parse :
+    latitude:((int32 * int32 * int32) * bool)
+    -> longitude:((int32 * int32 * int32) * bool)
+    -> altitude:int64
+    -> precision:(int64 * int64 * int64)
+    -> t
+  (** [parse ~latitude ~longitude ~altitude ~precision] Parse a human-readable format
+     to a Loc record.
+
+     [latitude] is represented by
+       ((degress, mintues, seconds), direction),
+     where direction is true for North and false for South.
```

Comment:
Thanks for your documentation. I'd find it slightly easier to use for direction a polymorphic variant, `` [ `North | `South ] `` (resp. `` [ `West | `East ] `` below). Or is that inconvenient in using this function?

Reply:
That sounds like a great idea. I will investigate implementing this.

Reply:
@hannesm, I've added this now!